mythril/laser/ethereum/keccak.py

```diff
@@ -16,7 +17,9 @@ def is_keccak(self, expression: Expression) -> bool:
         :param expression:
         :return:
         """
-        return str(expression) in self.keccak_expression_mapping.keys()
+        if not isinstance(expression, BitVecFunc):
+            return False
+        return expression.func_name == "keccak256"
```

Comment:
why keep the keccak function manager when we start using bitvecfuncs?

Reply:
It seemed to be a nice, handy and extensible class to other functions.

Reply:
What functionality does it add that BitVecFuncs do not already give?

Reply:
save some lines of code + we can extend this to ECDSA.

Reply:
If these are useful helper functions for bitvecfuns then why not add them as such instead of maintaining this class?

Reply:
My plan was to place few static classes in this file, so I left it off as it is.Will clean it up a bit.